### ISSUE TITLE
Would it be useful to be able to fill pdfs through an API?

### ISSUE THREAD

**Issue by bengolder** [department-of-veterans-affairs/hacktheforms#1]
I'm excited about this event but won't be able to attend. But I've been working with others at [Code for San Francisco](http://codeforsanfrancisco.org/) on a tool for turning PDFs into PDF-filling APIs: [pdfhook](https://github.com/codeforamerica/pdfhook)

We have [an example deployment](https://pdfhook.herokuapp.com/) here that would allow you to upload a form and create an API that fills the form. For example, here's a quickly-built API for filling VBA-21-0781-ARE.pdf: https://pdfhook.herokuapp.com/23/ 

Would this tool be of use at the event?

**Comment by akippolito** [comment]
Hello! 
THIS IS AMAZING! This is so helpful. We will share this with the teams today! :)

Can we be in touch to work together in the future?

**Comment by bengolder** [comment]
@akippolito absolutely! I was at [another hackathon yesterday](http://www.openlawlab.com/2016/04/25/hack-for-justice-event-legal-design-and-the-criminal-justice-system/) where I worked on another form-filling project.

My email is bgolder@codeforamerica.org
